### PR TITLE
Correct test and project structure section in the contributing docs

### PR DESCRIPTION
**Pull request by davidemarcoli**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] I have updated the documentation accordingly.
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] I have updated the "upcoming" section inside _docs/changelog.md_ explaining the changes I contributed

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Six Webcomponents Contributing Guide
+# SIX Web Components Contributing Guide
 
-Hi! I'm really excited that you are interested in contributing to the SIX Webcomponents. Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
+Hi! I'm really excited that you are interested in contributing to the SIX Web Components. Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
```
Comment (colinscz) [inline .github/contributing.md:3]:
```suggestion
Hi! We are really excited that you are interested in contributing to the SIX Web Components. Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
```

File: .github/contributing.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Hi! I'm really excited that you are interested in contributing to the SIX Webcom
 
 - Make sure tests pass!
 
-- Commit messages must follow the [commit message convention](./commit-convention.md) so that changelogs can be automatically generated. Commit messages are automatically validated before commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [simple-git-hooks](https://github.com/toplenboren/simple-git-hooks)).
+- Update the "upcoming" section inside _docs/changelog.md_ with the details of your changes.
```
Comment (colinscz) [inline .github/contributing.md:36]: Maybe mention the changelog we follow, I think Iwan has mentioned that somewhere with a link.
@ibirrer maybe you can quickly post it here?

File: .github/contributing.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ Hi! I'm really excited that you are interested in contributing to the SIX Webcom
 
 You will need [Node.js](https://nodejs.org) **version 16+**, and [NPM](https://www.npmjs.com/package/npm) **version 8**.
 
-After cloning the repo, run in root directory::
+After cloning the repo, run in root directory:
```
Comment (colinscz) [inline .github/contributing.md:54]:
```suggestion
After cloning the repository, run in root directory:
```

**Review by colinscz — CHANGES_REQUESTED** [review]

File: .github/contributing.md (hunk 2 above)
Comment (stefanrufer) [reply .github/contributing.md:36]: Link to https://keepachangelog.com/en/1.1.0/

**Review by colinscz — APPROVED** [review]

**Pull request merged**